GitUI/UserControls/FileStatusList.cs

```diff
@@ -1518,6 +1538,12 @@ public override int Compare(ListViewItem x, ListViewItem y)
 
         #endregion
 
+        #region private Color constants
+        //// Do not declare the colors "static" because Color.FromArgb() will not work at their initialization.
```

Comment:
nitpick: //// is intended for commenting out code (this "feature" is not used now though)

Reply:
Yes. Though with only //, the StyleCop scolds "Single-line comment should be preceded by blank line". And I didn't want a blank line after #region here. (The GE code uses both: #region with and without a following blank line.)